x-pack/plugins/security_solution/public/management/pages/event_filters/service/api_client.ts

```diff
@@ -55,9 +55,9 @@ export class EventFiltersApiClient extends ExceptionsListApiClient {
    */
   async getSuggestions(body: EndpointSuggestionsBody): Promise<string[]> {
     const result: string[] = await this.getHttp().post(
-      resolvePathVariables(SUGGESTIONS_ROUTE, { suggestion_type: 'eventFilters' }),
+      resolvePathVariables(SUGGESTIONS_INTERNAL_ROUTE, { suggestion_type: 'eventFilters' }),
```

Comment:
Could you add a test for this class that validate `getSuggestions()` calls the correct API?

Reply:
Added 
https://github.com/elastic/kibana/pull/194832/commits/93e9fe8fde3e5b5c7bee752325c68b52c78548ca